procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -22,13 +22,17 @@
     "import time\n",
```

Comment:
<p>You call this same block of code 4 times.</p><p>Best to make it a function.</p><p>I'd suggest using target 1 to cycle through all filters, with grating=empty,</p><p>then target 2 can be all filters with grating = hologram.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Now it's only called twice so I have left it as it was, so users can see also the code easier. </p>

Reply:
<p>I have implemented images in all instrument setups. Thanks!</p><p> </p>